Cargo.toml

```diff
@@ -21,14 +21,14 @@ branch = "otak/oxidize-wasm-v2"
 
 [patch.crates-io.openmls]
 git = "https://github.com/wireapp/openmls"
-branch = "feat/async"
```

Comment:
Since your PR there is already approved we can revert it to the async branch, right?